extensions/wikia/Recirculation/RecirculationHooks.class.php

```diff
@@ -123,7 +122,7 @@ function () {
 	}
 
 	private static function addLiftIgniterMetadata( OutputPage $outputPage ) {
-		global $wgDevelEnvironment, $wgLanguageCode, $wgStagingEnvironment;
+		global $wgDevelEnvironment, $wgLanguageCode, $wgStagingEnvironment, $wgIsPrivateWiki, $wgCityId;;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1116)